tests/Unit/Migrations/PermissionsMigrationTest.php

```diff
@@ -16,6 +17,14 @@ class PermissionsMigrationTest extends TestCase
     public function permissions_seeded()
     {
         $this->assertTrue(Schema::hasTable('permissions'));
-        $this->assertDatabaseCount('permissions', 4);
+
+        $seededPermissions = app(Permission::class)->whereIn('name', [
```

Comment:
@drewroberts I changed the way to test permissions_seeded here because now we require more packages in this package so the `permissions` table will have more than 4 permissions.